cpp/order/Steinhardt.h

```diff
@@ -152,12 +160,13 @@ class Steinhardt
                                   const freud::locality::NeighborQuery* points, freud::locality::QueryArgs qargs);
 
     //! Normalize the order parameter
-    float normalize();
+    float normalizeSystem();
```

Comment:
This is a separate issue, unrelated to Wl normalization. @bdice will make an issue describing this.

Reply:
I added a more descriptive comment instead of filing an issue. It's somewhat better. I'm not sure what references I would cite here but I'll keep it in mind if I see a paper defining that quantity.